### PR TITLE
Add mutation for update/create KYC fields for the user

### LINKED ISSUE #60: Add mutation to allow for existing users to update or fill in their KYC informations 25
Users will need to be able to fill in their KYC information at any time, we need to provide a GraphQL mutation for that

We must just make sure we have the right mutation depending on which if its a `company` or `person`

### PR DESCRIPTION
**Pull request by amiskov**
## What issue does this PR close
Closes #60 

## Changes Proposed (a list of new changes introduced by this PR)
Add mutations for creating and updating user KYC data and address.

Here's the suggested GraphQl API for both mutations. If the user has Address/KYC, then it will be updated; if not, then it will be created:

```graphql
mutation ($inputKyc: KycDataUpdateInput!, $inputAddress: AddressUpdateInput!) {
  upsertKyc(input: $inputKyc) {
    document
    #...
  }
  upsertAddress(input: $inputAddress) {    
    zip
    #...
  }
}

# Inputs
{
  "inputKyc": {
    "accountId": "andreymiskov",
    "countryId": "1",
    "phone": "44433333",
    "userType": "natural",
    "document": "111111111",
    "documentType": "cedula_de_identidad"
  },
  "inputAddress": {
    "accountId": "andreymiskov",
    "countryId": "1",
    "neighborhoodId": "1",
    "street": "Elm Street",
    "number": "321",
    "stateId": "1",
    "cityId": "1",
    "zip": "10102"
  }
}
```

## How to test (a list of instructions on how to test this PR)
- Run `mix test`.
- Try to run the query above in the GraphiQl interface.

### CODE REVIEW

File: lib/cambiatus_web/schema/kyc_types.ex [hunk 1]
```diff
@@ -47,4 +47,50 @@ defmodule CambiatusWeb.Schema.KycTypes do
...
+
+  object :kyc_mutations do
+    @desc "Updates user's KYC info if it already exists or creates a new one if user hasn't it yet."
+    field :update_or_create_kyc, :kyc_data do
+      arg(:input, non_null(:kyc_data_update_input))
+      resolve(&Kyc.update_or_create_kyc/3)
+    end
+
+    @desc "Updates user's address if it already exists or creates a new one if user hasn't it yet."
+    field :update_or_create_address, :address do
```
Comment (lucca65) [inline lib/cambiatus_web/schema/kyc_types.ex:91]: we could call this upsert, we already use this term in the contracts

**Review by lucca65 — CHANGES_REQUESTED** [review]

File: lib/cambiatus_web/schema/kyc_types.ex (hunk 1 above)
Comment (amiskov) [reply lib/cambiatus_web/schema/kyc_types.ex:91]: Oh, nice! Didn't know this word :)

Comment (lucca65) [reply lib/cambiatus_web/schema/kyc_types.ex:91]: made up dev word 😛

Comment (amiskov) [reply lib/cambiatus_web/schema/kyc_types.ex]: @lucca65 I decided to go with`upsert` as you suggested! We already use this word even in the frontend for Actions. The [changes](https://github.com/cambiatus/backend/pull/77/commits/343c738aaf4d2922bd7d99f33f4c5a6bdf65ccd7) were only in naming, so I decided not to ask you about re-reviewing and just merged this PR.

**Pull request merged**